docs/standard/modern-web-apps-azure-architecture/architectural-principles.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Architectural principles | Microsoft Docs 
+description: Architect Modern Web Applications with ASP.NET Core and Azure | Architectural principles
```

Comment:
add this to the SEO work item to update these too

Reply:
issue updated.